source/containers.tex

```diff
@@ -6331,8 +6331,8 @@
 The return value of \tcode{data()} is unspecified.
 
 \pnum
-The effect of calling \tcode{front()} or \tcode{back()} for a zero-sized array is
-undefined.
+The effect of calling \tcode{front()} or \tcode{back()}
+on a zero-sized array is undefined.
```

Comment:
Given LWG4276 is Tentatively Ready now, lets drop changes in this paragraph (which is to be removed).